### DISCUSSION TITLE
Tesla email addy changed

### DISCUSSION THREAD

**Discussion by rwskinner** [jasonacox/Powerwall-Dashboard#554] [General]
Tesla uses my email for the account name.  I changed it to my new email address.  After doing so, the PW Dashboard isn't updating battery info.  What do I need to do to modify the email address?

**Comment by rwskinner** [comment]
I just ran setup again.  I edited my email address in the config files but that didn't work.  I had to get a new access tokens.

**Reply by jasonacox** [reply]
You can request a new access token using this:

```bash
docker exec -it pypowerwall python3 -m pypowerwall setup
```

It will provide a URL for you to visit to log in to your Tesla account. The "404" page is expected, just copy the URL and paste it into the setup prompt as requested.

Restart the container:

```bash
docker restart pypowerwall
docker logs pypowerwall -f
```
